src/Serilog.Sinks.ElmahIO/Serilog.Sinks.ElmahIo.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <VersionPrefix>3.2.0</VersionPrefix>
+    <VersionPrefix>3.3.0</VersionPrefix>
     <Authors>Michiel van Oudheusden, elmah.io, Serilog Contributors</Authors>
     <TargetFrameworks>net45;netstandard1.3</TargetFrameworks>
```

Comment:
Might also be time to update the targets?

Reply:
Not sure I understand. Like in target frameworks?

Reply:
yeah net framework

Reply:
Not sure I want to target anything else. Serilog itself targets .NET 4.5 and up and netstandard 1.0 and up. netstandard 1.1 would be an option but haven't really received any request for supporting <1.3.

Reply:
Make sense, was just a thought. We recently did some changes in the Elasticsearch sink to drop support for older frameworks.

Reply:
Good point. Thought I was doing something wrong :D I will resolve this one then.